notebooks/create_synthetic_data.py

```
@@ -23,11 +23,20 @@
     raise RuntimeError(f"Failed to write synthetic data: {str(e)}")
+
+# COMMAND ----------
+synthetic_drift_df = generate_synthetic_data(config, full_data, drift = True)
+
+# COMMAND ----------
+try:
+    drift_table_path = f"{config.catalog}.{config.db_schema}.{config.use_case_name}_drift"
+    synthetic_drift_df.write.format("delta").mode("append").saveAsTable(drift_table_path)
+    print(f"Successfully appended synthetic data to {drift_table_path}")
+except Exception as e:
+    raise RuntimeError(f"Failed to write synthetic data: {str(e)}")
```

Ruff (0.7.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)